sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1525,7 +1525,7 @@ object SQLConf {
         "3. dapply " +
         "4. gapply " +
         "The following data types are unsupported: " +
-        "FloatType, BinaryType, ArrayType, StructType and MapType.")
+        "FloatType, ArrayType, and StructType.")
```

Comment:
SparkR Arrow optimization still won't work due to https://github.com/apache/spark/blob/93ea353cae0ccc187028f79024b239893ebaeb96/R/pkg/R/types.R#L87-L120

You can revert this change back. Let's do this separately in a separate PR.

Reply:
+1
Shall we revert the last two commits related to docs for now?

Reply:
Done